website/conferences/views.py

```diff
@@ -28,6 +29,22 @@
 
 logger = logging.getLogger(__name__)
 
+# Setup Keen client
+keen.project_id = settings.KEEN_PROJECT_ID
+keen.write_key = settings.KEEN_WRITE_KEY
+
+
+def _dt_to_iso8601(value):
```

Comment:
Would this work? https://github.com/CenterForOpenScience/osf.io/blob/8225f315d414e89dd72c0f277bca0f8a6823273e/framework/utils.py#L7-L9